### DISCUSSION TITLE
Question

### DISCUSSION THREAD

**Discussion by WSrujan** [5x/easy-steam-free-packages#31] [Q&A] [answered]
will this get my banned?

**Comment by WSrujan** [comment]
is possible run this on heroku, repl, files

**Comment by 5x** [comment, marked as answer]
>will this get my banned?

At the moment, no such case is known. The script simply emulates content "activation", which does not violate the licensing agreement in any way.

>is possible run this on heroku, repl, files

You can adapt this script for the Node.js environment and further use it on Heroku.

**Comment by WSrujan** [comment]
ok when will packages change again?
